### PR TITLE
Issue#206 Rearranged layout and added avatar slider

### PR DESCRIPTION
**Pull request by charlie0605**
Rearranged the layout of the avatar tool pellet 
added slider for the avatars
image
image

### CODE REVIEW

File: app/assets/javascripts/theatre.coffee [hunk 1]
```diff
@@ -1,7 +1,11 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
-backdropSlick = () ->
+init = () ->
+  slickSlider()
+  rangeSlider()
+
```
Comment (thongteav) [inline app/assets/javascripts/theatre.coffee]: Is this necessary?

**Comment by helenvarley** [comment]
looking good @charlie0605 :)  for the label on the show/hide name button, it doesn't need to have all that text. the word "avatar" definitely can go, since we know we're in the avatar palette. in UpStage the button only has the word "name" on it, but it's not totally intuitive so if there's room for "show/hide name" then that's fine.
or, perhaps better, it could have 2 states: when the name is visible, the button text reads "hide name" and when the name is hidden, it reads "show name". is that possible?

**Comment by charlie0605** [comment]
@helenvarley Yes that is possible.

**Comment by charlie0605** [comment]
@rohrlach is it ok to add the name toggle in this issue or should it be a separate issue?

File: app/assets/stylesheets/theatre.scss [hunk 2]
```diff
@@ -208,3 +197,18 @@ div.message > div.content.shout {
       width: 95%;
     }
 }
+
+.funButtons {
+  padding-top: 10px;
+  padding-bottom: 10px;
+}
+
+.buttonsClass{
```
Comment (rohrlach) [inline app/assets/stylesheets/theatre.scss]: Try to follow conventions - class names should be lowercase and hyphenated ```.fun-buttons``` also I'm not sure I like the names "funButtons" and "buttonsClass" they're really vague

File: app/views/theatre/performance.html.erb [hunk 3]
```diff
@@ -58,33 +58,34 @@
...
+            </div>
           </div>
-          <div class="range-slider">
-            Size
-            <input id = "avatarSlider" class = "range-slider-range" type="range" min="1" max="10" value = "5">
-            <span class = "range-slider-value">-</span>
+          <div class="funButtons row">
+            <div class="buttonsClass">
+              <button class="btn btn-default" id="avatarName" disabled="disabled">
+                Show/Hide Avatar Name
```
Comment (rohrlach) [inline app/views/theatre/performance.html.erb]: As @helenvarley mentioned - This should just say "Show/Hide Name"

Comment (charlie0605) [reply app/views/theatre/performance.html.erb]: she also mentioned to have it to toggle, can it be on the same issue or separate?

**Pull request merged**